addons/product/models/product_template.py

```diff
@@ -507,6 +507,19 @@ def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_ui
             '', args=[('id', 'in', list(searched_ids))],
             operator='ilike', limit=limit, name_get_uid=name_get_uid)
 
+    def action_open_label_layout(self):
+        view = self.env.ref('product.product_label_layout_form')
```

Comment:
```suggestion
        self.ensure_one()
        view = self.env.ref('product.product_label_layout_form')
```

Reply:
@svs-odoo why the `ensure_one`? there is even an `self.ids` below